### PR TITLE
fix: Unbreak Docker build with symbolicator-crash

### PR DESCRIPTION
**Pull request by Swatinem**
#skip-changelog

### CODE REVIEW

**Comment by loewenheim** [comment]
Why does the docker build currently break and why does this unbreak it?

**Comment by Swatinem** [comment]
The docker build currently fails with:

```
Step 12/19 : RUN mkdir -p crates/symbolicator/src     && echo "fn main() {}" > crates/symbolicator/src/main.rs     && cargo build --release
 ---> Running in 5458810100e2
error: failed to load manifest for workspace member `/work/crates/symbolicator`

Caused by:
  failed to load manifest for dependency `symbolicator-crash`

Caused by:
  failed to read `/work/crates/symbolicator-crash/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```

It tries to find a non-existant symbolic-crash. Because due to docker layer caching, we very selectively only copy very specific files into the docker image, and symbolic-crash was not one of them, hence it fails because it does not exist.

**Comment by loewenheim** [comment]
I see, thanks for the explanation. I don't know a whole lot about how docker works.

**Review by loewenheim — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -3,10 +3,14 @@ FROM debian:stretch-slim AS symbolicator-build
 
 WORKDIR /work
 
+# Hooray for running with a totally outdated debian image
```
Comment (kamilogorek) [inline Dockerfile:6]: Cant we update it instead? 👀

**Pull request merged**